src/app/shared/uploader/uploader.component.scss

```diff
@@ -0,0 +1,40 @@
+@import '../../../styles/_variables.scss';
+
+.ds-base-drop-zone {
+  border: 2px dashed $gray-600;
+}
+
+/* Default class applied to drop zones on over */
+.ds-base-drop-zone-file-over {
+  border: 2px dashed map-get($theme-colors, primary);
+}
+
+.ds-base-drop-zone p {
+  height: 42px;
+}
+
+.ds-document-drop-zone {
+  top: 0;
+  left: 0;
+  z-index: -1;
+}
+
+.ds-document-drop-zone-active {
+  z-index: 1025 !important;
```

Comment:
If these z-indexes can't use one of bootstrap's standard values, best move them to `_custom_variables.scss` so we can get an overview of all custom z-indexes in dspace, and prevent unexpected conflicts.